### ISSUE TITLE
Using dm-writeboost with lvmthin

### ISSUE THREAD

**Issue by justnx** [akiradeveloper/dm-writeboost#209]
Hi,

good project so far. I really like your approach.

I want you to ask if it's out of the box possible to use dm-writeboost in combination with lvm or lvmthin. I had no luck so far. So any hints for doing this would be very nice.

**Comment by akiradeveloper** [comment]
@justnx Yes, you can. You can use lvm devices as either underlying device and that's actually the way I test this driver.

**Comment by coinblockers** [comment]
Do i need to whitelist the device mapper path in lvm config or something?
I fail at adding /dev/mapper/cacheddevice into a pv or vg already or is this the wrong approach?
Can you give an example and maybe put that in your Wiki. That would be awesome.

**Comment by akiradeveloper** [comment]
I don't know what you guys are trying to do. LVM device is just a DM device that's managed by tool called LVM. Since stacking DM devices is valid. There is nothing you have to do any more.

This might help: dm-writeboost **can't** be managed by LVM and there is no plan to do so. Instead users manage dm-writeboost by other userland tools. All you can do is creating LVM devices and then create WB device using them and the opposite is impossible because LVM devices are created at boot time. So, it you are trying to stack some LVM device (like dm-thin) **above** WB device this would be failure because the WB device is no where to be seen at the momemt.

**Comment by justnx** [comment]
Hmm i'm wondering how is lvmcache with dm-cache doing this. 
Cant we just use dm-writeboost as an replacement of dm-cache for such purposes?

**Comment by bash99** [comment]
@akiradeveloper  dm-writeboost under lvm-thin can be used in KVM host, all vm can have a performance-boost thin provisioning storage. otherwise you have to set it up one by one.
Another choice is use qcow2 format on dm-writeboost backed file-system, but it's not as fast as raw device on lvm-thin.
